Tribler/Test/test_torrent_store_plyvel.py

```diff
@@ -0,0 +1,74 @@
+# test_torrent_store.py ---
+#
+# Filename: test_torrent_store.py
+# Description:
+# Author: Elric Milon
+# Maintainer:
+# Created: Wed Jan 21 12:45:30 2015 (+0100)
+
+# Commentary:
+#
+#
+#
+#
+
+# Change Log:
+#
+#
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+# Code:
+from twisted.internet.task import Clock
+
+from tempfile import mkdtemp
+
+from Tribler.Core.torrentstore import TorrentStore
+from Tribler.Core.plyveladapter import LevelDB
+from Tribler.Test.test_torrent_store import TestTorrentStore
+
+
+K = "foo"
+V = "bar"
+
+
+class ClockedTorrentStore(TorrentStore):
+    _reactor = Clock()
+
+    def __init__(self, store_dir):
+        super(TorrentStore, self).__init__()
+
+        self._store_dir = store_dir
+        self._pending_torrents = {}
+        self._db = LevelDB(store_dir)
```

Comment:
Instead of copypasting the whole init just to override self._db, lets set the DB class as class attr. 
(like ```_reactor```)

here:
 https://github.com/Tribler/tribler/pull/1488/files#diff-81c3e3d81d31ae3aead8c67e57692af7L61

So it can easily be set on the test before instancing the class.